GDJS/Runtime/spriteruntimeobject.ts

```diff
@@ -1180,10 +1191,21 @@ namespace gdjs {
       if (this._animationFrameDirty) {
         this._updateAnimationFrame();
       }
-      const unscaledHeight = this._renderer.getUnscaledHeight();
-      if (unscaledHeight !== 0) {
-        this.setScaleY(newHeight / unscaledHeight);
+      this._setHeight(newHeight);
+    }
+
+    /**
+     * Change the size of the object.
+     *
+     * @param newWidth The new width of the object, in pixels.
+     * @param newHeight The new height of the object, in pixels.
+     */
+    setSize(newWidth: float, newHeight: float): void {
+      if (this._animationFrameDirty) {
+        this._updateAnimationFrame();
       }
+      this._setWidth(newWidth);
+      this._setHeight(newHeight);
     }
```

Comment:
I think it's not necessary to define 2 methods only to avoid 2 conditions. The frame will be updated only once either way.

Reply:
Never mind what I said, I look it too quickly.

Reply:
Agree that if we can avoid _setWidth/_setHeight it's better.

From the point of view of the reader, you don't immediately understand "why are there two methods with the same name, but one with an underscore, i.e: private? Is it public or private?"

I think here we can keep just a single pair: setWidth/setHeight and call them both

Reply:
Maybe the code is small enough to be inlined?